server/src/services/index.js

```diff
@@ -16,21 +13,11 @@ const getZipCode = (cllString) => {
 
 const services = {
   handleYelpSearch: (req, res) => {
-    const baseUrl = 'https://api.yelp.com/v2/search/';
 
     const defaultParams = {
-      sort: '1', // 0=Best matched (default), 1=Distance, 2=Highest Rated
+      sort_by: 'distance', // 0=Best matched (default), 1=Distance, 2=Highest Rated
```

Comment:
Are these changes a result of changes to the Yelp API?

Reply:
yes this is because of the v3 api

Reply:
https://www.yelp.com/developers/documentation/v3/business_search